http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/AppRoutes.scala

```diff
@@ -176,8 +177,6 @@ class AppRoutes(kubernetesService: AppService[IO], userInfoDirectives: UserInfoD
       apiCall = kubernetesService.deleteApp(
         deleteParams
       )
-      tags = Map("deleteDisk" -> deleteDisk.toString)
```

Comment:
`appType` isn't available here, so moving where we're adding metrics to the interp